docs/notebooks/failure_ego.pct.py

```diff
@@ -49,12 +52,14 @@
 # %%
 def masked_branin(x):
     x0 = np.abs(x[:, 0] - 0.5) < 0.25
-    x1 = np.abs(x[:, 1] - 0.5) < 0.40
-    mask_nan = np.logical_and(x0, x1)
+    x1 = np.abs(x[:, 1] - 0.5) < 0.140
+    mask_nan = np.sqrt((x[:, 0] - 0.5) ** 2 + (x[:, 1] - .4) ** 2) < 0.3
+    # mask_nan = np.logical_and(x0, x1)
```

Comment:
what's happening here?

Reply:
forgot to clean, sorry, only l56 is relevant